stanford-earth/matson
# 8.x 1.x earth 944 required webform

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/EARTH-944_
This adds the little red asterisks as a background image.

# Needed By (Date)
- Soon

# Urgency
- Fixes broken stuff

# Steps to Test

- Checkout this branch
- Clear cache
- Navigate to __form/contact__
- Verify you see the little red asterisks 

# Affected Projects or Products
- Earth (https://earth.stanford.edu)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/EARTH-944

## Related PRs

## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
